hcatalog/core/src/test/java/org/apache/hive/hcatalog/mapreduce/HCatMapReduceTest.java

```diff
@@ -282,7 +282,7 @@ public void map(WritableComparable key, HCatRecord value, Context context)
   Job runMRCreate(Map<String, String> partitionValues, List<HCatFieldSchema> partitionColumns,
       List<HCatRecord> records, int writeCount, boolean assertWrite) throws Exception {
     return runMRCreate(partitionValues, partitionColumns, records, writeCount, assertWrite,
-        true, null);
+        true, new HashMap<String, String>());
```

Comment:
Can `Collections.newMap()` be used instead?

Reply:
Changed.